static/src/js/components/GranuleResults/__tests__/GranuleResultsDataLinksButton.test.js

```diff
@@ -76,6 +77,9 @@ describe('GranuleResultsDataLinksButton component', () => {
 
   describe('with multiple granule links', () => {
     test('renders the correct element', () => {
+      // Mocks createPortal method of ReactDOM (https://stackoverflow.com/a/60953708/8116576)
+      ReactDOM.createPortal = jest.fn(modal => modal);
```

Comment:
It's strange to use `modal` in this case. I would prefer to see this named `dropdown` or something more specific to this implementation.

Reply:
Yes, right! I'll make that change. 

Also, we will have to write a test that checks if dropdown opens when it is clicked from table view. (This test only checks if it opens when there are more links). I'll add that test as well then we can merge I guess.

Reply:
Yes, that is correct. And once this is done, it'll be ready to merge!

Reply:
Fixed this and added a test for CustomDataLinksToggle component